cosmos/azure-spring-data-cosmos/cosmos-multi-tenant-samples/tenant-by-container/src/main/java/com/azure/spring/data/cosmos/example/OrderRepository.java

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.spring.data.cosmos.example;
+
+import com.azure.spring.data.cosmos.repository.CosmosRepository;
+import com.azure.spring.data.cosmos.repository.Query;
+import com.azure.spring.data.cosmos.repository.ReactiveCosmosRepository;
+import org.springframework.data.repository.query.Param;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+
+@Repository
+public interface OrderRepository extends CosmosRepository<Order, String> {
+    @Query(value = "SELECT c.id, c.orderDetail, c.lastName FROM c where c.type = 'order'")
```

Comment:
1. Yes
2. Yes
3. Yes, we need to show it, because entity co-location is one of the two benefits of choosing container-level isolation instead of database level isolation with multi-tenancy (the other being the ability to use shared throughput model).